internal/addon/manifests/charts/mcoa/charts/tracing/values.yaml

```diff
@@ -1,2 +1,3 @@
 nameOverride: null
-enabled: true
\ No newline at end of file
+enabled: true
+instrumentationEnabled: true
```

Comment:
Do you think we should also add `collectorEnabled: true` for the use cases where users only want to deploy instrumentation?

Reply:
Indeed but `enabled` is also connected with the installation of other resources like the opentelemetry-operator. So in this case, if you want to have the ability for users to deploy both `OpenTelemtryCollector` and `Instrumentation` independently then you should have two separate `enable` flags or you might also be able to change the `if` clause to check if either `otelColSpec`  or `instrumentationSpec`  and different than an empty string.

Reply:
For now, I think it is good enough to have instrumentation only if you enable the creation of the OpenTelemetry Collector. We can change this behavior later if we find it is a real use case.